src/client/messages.js

```diff
@@ -1,5 +1,8 @@
 export default {
   en: {
+    app: {
```

Comment:
Basically order is ensured, but it's not realiable across different serialization and deserialization libraries I guess. So for this case, I'm suggesting gold rule: Favor explicit over smart.